### ISSUE TITLE
How to get gojsonschema in generate.sh 

### ISSUE THREAD

**Issue by echo567** [bsquizz/crd-codegen#4]
hello,
![image](https://github.com/RedHatInsights/crd-codegen/assets/33777600/61fffc10-8e96-496d-aa54-c1913498d9ab)
i found this  https://github.com/xeipuuv/gojsonschema.  But it cannot be compiled into a command
Hope to reply, THANK！

**Comment by reedjosh** [comment]
Pretty sure this tool is/was the one: https://github.com/omissis/go-jsonschema

Similar options and provides the functionality required there.

It does install as go-jsonschema though. I'm guessing the validation tool got the gojsonschema name in place of the generation tool.

**Comment by bsquizz** [comment]
Yes, thanks! It looks like github.com/atombender/go-jsonschema now redirects to https://github.com/omissis/go-jsonschema
